src/Simplex/Chat.hs

```diff
@@ -1688,6 +1693,8 @@ processChatCommand = \case
         (connId, cReq) <- withAgent $ \a -> createConnection a (aUserId user) True SCMInvitation Nothing subMode
         -- [incognito] reuse membership incognito profile
         ct <- withStore' $ \db -> createMemberContact db user connId cReq g m mConn subMode
+        -- TODO not sure it is correct to set connections status here?
```

Comment:
I think we did it in UI as a workaround for the fact it's not separately subscribed to avoid spinning circle on this contact